helmfile.d/20-appserver.yaml

```diff
@@ -0,0 +1,26 @@
+bases:
+- ../environments.yaml
+  
+---
+
+helmDefaults:
+  atomic: true
+  timeout: 6000
```

Comment:
This is 100 minutes… if possible the pod should become active within 3 minutes, then the timeout should not exceed that value. If it takes longer than that to do a health check in the first place, consider decreasing the liveness and readiness probe intervals.